### PR TITLE
[Container app] Release version `0.3.52`

### PR DESCRIPTION
**Pull request by Greedygre**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Container app

**Comment by Greedygre** [comment]
Wait for PR: https://github.com/Azure/azure-cli-extensions/pull/7643

File: src/containerapp/HISTORY.rst [hunk 1]
```diff
@@ -4,13 +4,20 @@ Release History
 ===============
 upcoming
 ++++++
+
+0.3.52
+++++++
+* 'az containerapp env java-component admin-for-spring': Support create/update/show/delete Admin for spring
+* 'az containerapp env java-component nacos': Support create/update/show/delete Nacos
+* 'az containerapp env java-component': Support --bind, --unbind to bind or unbind with other Java Components
```
Comment (Greedygre) [inline src/containerapp/HISTORY.rst:12]: These command are supported from PR: https://github.com/Azure/azure-cli-extensions/pull/7622
It's history added to wrong version `0.3.51`, fixed.

File: src/containerapp/HISTORY.rst [hunk 2]
```diff
@@ -4,13 +4,20 @@ Release History
...
 * 'az containerapp create/update': Update default value of enableMetrics to True for java runtime setting
 * 'az containerapp sessionpool create': Fix default container name to be lower case letters
 * 'az containerapp create/update': Support --scale-rule-identity for scale rule to authenticate to azure resource scaler
 * 'az containerapp sessionpool create/update': Fix sessionpool registry server fieldname and appending of registry secrets.
 
 0.3.51
 ++++++
+* Upgrade api-version to 2024-02-02-preview
```
Comment (Greedygre) [inline src/containerapp/HISTORY.rst:20]: In `0.3.51`, api-version upgraded to 2024-02-02-preview, add history.

**Review by Juliehzl — APPROVED** [review]

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**